linux/ibus-keyman/tests/scripts/run-tests.sh

```diff
@@ -57,68 +52,22 @@ function run_tests() {
   DISPLAY_SERVER=$1
   shift
 
-  echo > "$PID_FILE"
-  TEMP_DATA_DIR=$(mktemp --directory)
-  echo "rm -rf ${TEMP_DATA_DIR} || true" >> "$PID_FILE"
-
-  COMMON_ARCH_DIR=
-  [ -d "${TOP_SRCDIR}"/../../core/build/arch ] && COMMON_ARCH_DIR=${TOP_SRCDIR}/../../core/build/arch
-
-  if [ -d "${COMMON_ARCH_DIR}"/release ]; then
-    COMMON_ARCH_DIR=${COMMON_ARCH_DIR}/release
-  elif [ -d "${COMMON_ARCH_DIR}"/debug ]; then
-    COMMON_ARCH_DIR=${COMMON_ARCH_DIR}/debug
+  if [ -d "$(dirname "$0")/../../../build/$(arch)/debug" ]; then
+    CONFIG=debug
+  elif [ -d "$(dirname "$0")/../../../build/$(arch)/release" ]; then
+    CONFIG=release
   else
-    echo "Can't find neither ${COMMON_ARCH_DIR}/release nor ${COMMON_ARCH_DIR}/debug"
-    exit 2
+    echo "Can't find neither ../../../build/$(arch)/debug nor ../../../build/$(arch)/release"
```

Comment:
```suggestion
    echo "Cannot find ../../../build/$(arch)/debug or ../../../build/$(arch)/release"
```

Yay English grammar (or ... "Can find neither a nor b" is technically okay but weird)